sonar-plugin-api/src/main/java/org/sonar/api/server/ws/internal/ValidatingRequest.java

```diff
@@ -49,6 +51,15 @@ public void setAction(WebService.Action action) {
   }
 
   @Override
+  public LocalConnector getLocalConnector() {
```

Comment:
what about that point, @simonbrandhof ?

Do you think we can ignore it because, even though this class is in the API, it is in an internal package?